seqr/views/utils/orm_to_json_utils.py

```diff
@@ -754,10 +754,10 @@ def get_json_for_matchmaker_submission(submission):
         additional_model_fields=['contact_name', 'contact_href', 'submission_id'])
 
 
-def get_json_for_rna_seq_outliers(models, **kwargs):
+def get_json_for_rna_seq_outliers(models, is_significant=True, **kwargs):
```

Comment:
It works for both expression and splicing outliers. But we don't have a significant threshold for splicing outliers which I think they should have.

Reply:
Per https://github.com/broadinstitute/seqr/issues/3200#issuecomment-1433752184

> not more than the top 50 splice junctions (based on sorting by pValue) and where pValue < 0.01